labs/src/main/scala/org/scalalabs/basic/lab01/OOExercise.scala

```diff
@@ -40,6 +40,32 @@ import scala.language.implicitConversions
  *   of type [[org.scalalabs.basic.lab01.CurrencyConverter]]
  * - Use the implicit CurrencyConverter to do the conversion. 
  */
-class Euro {
+class Euro (var euro:Int, var cents:Int, symbol:String) extends Currency (symbol){
+  var symbol = "E"
+  def this(e:Int, c:Int) = this(e, c, "E")
+  def this(e:Int) = this(e, 0, "E")
```

Comment:
This is a very Java-esque way of declaring different constructors. I bumped into some sources which suggested

```
class MyClass(val field1: String, val field2: Int = 5) { ... }
```

instead